brave-cxf/src/main/java/com/github/kristofa/brave/cxf/ServerRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.github.kristofa.brave.cxf;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.apache.cxf.message.Message;
+
+import com.github.kristofa.brave.http.HttpServerRequest;
+
+/**
+ * @author Michał Podsiedzik
+ */
+public class ServerRequest extends MessageWrapper implements HttpServerRequest {
```

Comment:
please make classes like this package private final. they aren't important enough to be public